reconcile/test/test_slack_usergroups.py

```diff
@@ -86,6 +89,35 @@ def test_get_pagerduty_username_slack_username(self):
         result = integ.get_pagerduty_name(user)
         self.assertEqual(result, 'pd')
 
+    @patch.object(integ, 'SlackApi')
```

Comment:
Same comment on `autospec` - I won't add comments for other occurrences, but please add it for all of them unless there's a good reason not to.